code/modules/mob/living/carbon/human/human_species.dm

```diff
@@ -9,6 +9,7 @@
 	living_mob_list -= src
 	dead_mob_list -= src
 	delete_inventory()
+	update_icons_body()
```

Comment:
This is already done, though, in human/New().

human/New() -> set_species() -> regenerate_icons() -> update_icons_body()

Reply:
Is it guaranteed to spawn without species set at all as the human/New() seems to only set species if there is none.

Reply:
Yes, it is. Species is null by default on humans. You would have to have a subtype of /mob/living/carbon/human with a New() that digs into all_species, sets species to a reference of the datum, etc.

No such thing exists, and mannequins definitely have set_species() called on them. How else would they appear as your species?

Reply:
Fair enough. Gonna have to dig into the stuff a bit deeper.

Reply:
I mean, it's a byond bug. No matter what you do, you may not be able to fix this. Assuming you can is folly.

You should make sure you're using the latest beta, and collect crash dumps to give to Lummox for troubleshooting. And make test cases for him if you can.